src/db/options.rs

```diff
@@ -284,25 +284,6 @@ pub struct ListCollectionsOptions {
     pub comment: Option<Bson>,
 }
 
-/// Specifies the options to a
```

Comment:
I moved this to `operation/list_databases.rs` since it's now describing what options the server expects in the operation, not part of the public API.